ext/spl/spl_observer.stub.php

```diff
@@ -4,155 +4,154 @@
 
 interface SplObserver
 {
-    /** @return void */
-    public function update(SplSubject $subject);
+    public function update(SplSubject $subject): void;
```

Comment:
Should be tentative?

Reply:
Definitely! Fix is on the way with the rebase